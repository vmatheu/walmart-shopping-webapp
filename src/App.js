import React from 'react';
import { SearchProduct } from 'components/SearchProduct';

const App = () => (
  <SearchProduct />
);

export default App;
