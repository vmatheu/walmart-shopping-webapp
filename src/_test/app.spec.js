import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import App from 'App';

describe('<App/>', () => {
  it('should have the same ui structure', () => {
    const wrapper = shallow(<App loading />);
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
  });
});
