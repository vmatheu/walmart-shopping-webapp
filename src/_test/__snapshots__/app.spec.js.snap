// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> should have the same ui structure 1`] = `<SearchProduct />`;
